### PR TITLE
youtube-dl: add quotation marks

### PR DESCRIPTION
**Pull request by Atrate**
Adding quotation marks when specifying a URL is necessary, since youtube URLs almost always contain a question mark (?), which is usually interpreted by the shell as globbing.

Signed-off-by: Atrate <Atrate@protonmail.com>

- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/youtube-dl.md [hunk 1]
```diff
@@ -5,15 +5,15 @@
 
 - Download a video or playlist:
 
-`youtube-dl {{https://www.youtube.com/watch?v=oHg5SJYRHA0}}`
+`youtube-dl {{'https://www.youtube.com/watch?v=oHg5SJYRHA0'}}`
```
Comment (navarroaxel) [inline pages/common/youtube-dl.md:8]: Based on #4363 token should be outside the `{}`
 ```suggestion
`youtube-dl '{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}'`
```

File: pages/common/youtube-dl.md [hunk 2]
```diff
@@ -5,15 +5,15 @@
...
 
 - List all formats that a video or playlist is available in:
 
-`youtube-dl --list-formats {{https://www.youtube.com/watch?v=Mwa0_nE9H7A}}`
+`youtube-dl --list-formats {{'https://www.youtube.com/watch?v=Mwa0_nE9H7A'}}`
```
Comment (navarroaxel) [inline pages/common/youtube-dl.md:12]:
```suggestion
`youtube-dl --list-formats '{{https://www.youtube.com/watch?v=Mwa0_nE9H7A}}'`
```

File: pages/common/youtube-dl.md [hunk 3]
```diff
@@ -5,15 +5,15 @@
...
 
 - Download a video or playlist at a specific quality:
 
-`youtube-dl --format "{{best[height<=480]}}" {{https://www.youtube.com/watch?v=oHg5SJYRHA0}}`
+`youtube-dl --format "{{best[height<=480]}}" {{'https://www.youtube.com/watch?v=oHg5SJYRHA0'}}`
```
Comment (navarroaxel) [inline pages/common/youtube-dl.md:16]:
```suggestion
`youtube-dl --format "{{best[height<=480]}}" '{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}'`
```

File: pages/common/youtube-dl.md [hunk 4]
```diff
@@ -29,7 +29,7 @@
 
 - Download a particular language's subtitles along with the video:
 
-`youtube-dl --sub-lang {{en}} --write-sub {{https://www.youtube.com/watch?v=Mwa0_nE9H7A}}`
+`youtube-dl --sub-lang {{en}} --write-sub {{'https://www.youtube.com/watch?v=Mwa0_nE9H7A'}}`
```
Comment (navarroaxel) [inline pages/common/youtube-dl.md:32]:
```suggestion
`youtube-dl --sub-lang {{en}} --write-sub '{{https://www.youtube.com/watch?v=Mwa0_nE9H7A}}'`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Just a few comments

**Review by navarroaxel — CHANGES_REQUESTED** [review]
The `{{url}}` in the other examples should be between `'` then.

**Review by sbrl — APPROVED** [review]
Thanks, @Atrate! Looks ok to me :-)

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**